cmd/kogito/command/context/root.go

```diff
@@ -73,6 +72,6 @@ func (i *rootCommand) InitHook() {
 	i.command.PersistentFlags().StringVar(&i.flags.cfgFile, "config", "", "config file (default is $HOME/.kogito/config.yaml)")
 	i.command.PersistentFlags().BoolVarP(&logVerbose, "verbose", "v", false, "verbose output")
 	i.command.PersistentFlags().Bool("version", false, "display version")
-	i.command.Version = version.Version
+	i.command.Version = "0.5.1"
```

Comment:
why exactly are you hardconding the version?

Reply:
To not mess around in other places. This is a config management problem actually. We should be able to release the CLI/operator images individually.

Reply:
Shall we change `version.Version` back to `0.5.0`?